lib/rules/__tests__/index.test.mjs

```diff
@@ -74,6 +76,25 @@ describe('custom message option', () => {
 	);
 });
 
+describe('invalid options', () => {
+	test.each(ruleNames)('"%s" should handle invalid options', async (name) => {
+		const rule = await rules[name];
+		const primaryOption = false;
```

Comment:
note

We have a related test named "with options (boolean)" in `validateOptions.test.mjs`.